Library/Homebrew/formula.rb

```diff
@@ -980,7 +998,7 @@ def migration_needed?
   end
 
   # @private
-  def outdated_versions
+  def outdated_versions(options = {})
     @outdated_versions ||= begin
```

Comment:
I wonder if it is still appropriate to cache the return value in a single instance variable, as the result can change based on the given `options`. I would expect the following sequence to produce different results, but it probably won't because the first one will have already populated the cache, so the options on the second invocation don't have any effect:

```ruby
f.outdated_versions
f.outdated_versions(:fetch_head => true)
```